src/components/LoginDropDownComponent/DropDownComponent.js

```diff
@@ -111,57 +116,69 @@ function DropdownComponent() {
                     leave="transition ease-in duration-75"
                     leaveFrom="transform opacity-100 scale-100"
                     leaveTo="transform opacity-0 scale-95">
-                    <Menu.Items className="absolute p-4 right-0 w-40 mt-2 origin-top-right bg-black/[0.9] text-white rounded-md shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none">
-                        {!isLoggedIn && (
-                            <Menu.Item
-                                as="div"
-                                onClick={() => {
-                                    openNewKeysModal();
-                                }}
-                                disabled={isLoggedIn}
-                                className="ui-active:bg-blue-500 ui-active:text-white ui-not-active:bg-white ui-not-active:text-black">
-                                Create Account
-                            </Menu.Item>
-                        )}
-                        {!isLoggedIn && (
-                            <Menu.Item
-                                as="div"
-                                disabled={isLoggedIn}
-                                onClick={() => {
-                                    openLoginModal();
-                                }}
-                                className="ui-active:bg-blue-500 ui-active:text-white ui-not-active:bg-white ui-not-active:text-black">
-                                Login
-                            </Menu.Item>
-                        )}
+                    <Menu.Items className="absolute right-0 z-10 mt-2 w-48 origin-top-right bg-white rounded-md shadow-lg ring-1 ring-black ring-opacity-5 focus:outline-none">
+                        <div className="py-1">
+                            {!isLoggedIn ? (
+                                <>
+                                    <Menu.Item>
+                                        {({ active }) => (
+                                            <button
+                                                onClick={openNewKeysModal}
+                                                className={`${
+                                                    active
+                                                        ? 'font-semibold'
+                                                        : 'font-normal'
+                                                } group flex w-full items-center rounded-md px-2 py-2 text-sm text-gray-700 hover:text-gray-900`}
+                                                disabled={isLoggedIn}>
+                                                Create Account
```

Comment:
Think we can add some svg here on login,create_account and logout button? It'd give more personalized experience.

Reply:
added the svg buttons please review